### DISCUSSION TITLE
Hello and question about port security

### DISCUSSION THREAD

**Discussion by akvadrako** [neutralinojs/neutralinojs#392] [General]
Hello. I just started looking at this project after evaluating NW, NodeGUI, webview (go), sciter, DeskGap and pywebview. So far it looks like the best of them: it didn't crash on startup, the web inspector works and it supports Wayland.

I would mention the Wayland support on your homepage since it's an important feature even Electron doesn't have (in released versions).

I have a question about security though – when I start a neu app I can connect to the open port from other users on the system. Wouldn't it make sense to build in some kind of auto-generated auth token so by default apps are secure against local attacks?

**Comment by shalithasuranga** [comment]
If we use the `cloud` mode, anyone from the local network can access the application. Otherwise, only the current computer has access to the port. Also, Neutralino uses a simple auto-generated token to prevent malicious requests. But, we can obtain the token by loading the application. Do you mean other users who are logged into the computer can trigger native calls via Neutralino port right?

**Reply by akvadrako** [reply]
I didn't see the auth token before, though I'm not sure how it functions. What I would expect is that if I run an app, other local users cannot access it.
